index.src.html

```diff
@@ -2478,7 +2483,33 @@
         |init|.{{AudioDataInit/data}}.
     8. Let |resourceReference| be a reference to |resource|.
     9. Assign |resourceReference| to {{AudioData/[[resource reference]]}}.
-3. Return |frame|.
+3. Return |audio|.
+
+<dfn constructor for=AudioData title="AudioData(source, init)">
+  AudioData(source, init)
+</dfn>
+1. If |source|.{{platform object/[[Detached]]}} is `true`, throw an
+    {{InvalidStateError}} {{DOMException}}.
+2. If |init|.{{AudioDataCopyInit/timestamp}} does not [=map/exist=],
```

Comment:
Using `clone()` here means we do a shallow copy instead of a deep copy.

I think that this is ultimately desirable, but it might be unexpected or confusing. Should we always make a deep copy?

Reply:
Can we simplify this so that there is there is a single path? Override with `AudioDataCopyInit/timestamp` if passed in, use the source `timestamp` if not.